integrations/pgvector/src/haystack_integrations/document_stores/pgvector/document_store.py

```diff
@@ -475,6 +483,56 @@ def delete_documents(self, document_ids: List[str]) -> None:
 
         self._execute_sql(delete_sql, error_msg="Could not delete documents from PgvectorDocumentStore")
 
+    def _keyword_retrieval(
+        self,
+        user_query: str,
+        top_k: int = 10,
+        filters: Optional[Dict[str, Any]] = None,
+        language: Optional[str] = "english",
```

Comment:
```suggestion
```

I would remove this parameter.
Once the language is set, you cannot search using other languages.
Am I wrong?